apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -113,9 +113,10 @@ def prepare(element_html, data):
                         for buttons in groups:
                             if buttons.tag == "pl-drawing-button":
                                 type_name = buttons.attrib.get("type", None)
-                                if type_name == "pl-arc-vector-CCW":
-                                    type_name = "pl-arc-vector"
-                                elif type_name == "pl-arc-vector-CW":
+                                if (
+                                    type_name == "pl-arc-vector-CCW"
+                                    or type_name == "pl-arc-vector-CW"
```

Comment:
This would be resolved by the `pylint` ruff rule: https://docs.astral.sh/ruff/rules/repeated-equality-comparison/#repeated-equality-comparison-plr1714, which is not split up into a separate PR yet (still in #11056 )